test/e2e/manifest/testVPC/customize_networkconfig.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: nsx.vmware.com/v1alpha1
+kind: VPCNetworkConfiguration
+metadata:
+  name: selfdefinedconfig
+spec:
+  defaultGatewayPath: /infra/tier-0s/PLR
+  edgeClusterPath: /infra/sites/default/enforcement-points/default/edge-clusters/{}
+  defaultIPv4SubnetSize: 26
+  nsxtProject: nsx_operator_e2e_test
+  externalIPv4Blocks:
+    - ipblock-public-for-vpc
```

Comment:
During testbed deployment, we already create a external project for nsx_operator_e2e_test

>            "display_name": "e2e_test_external_ip_blk",
 >           "description": "nsx_operator_e2e_test project external ip block",
  >          "visibility": "EXTERNAL",
   >         "cidr": "172.160.0.0/16"

Shall we use this external IP blk in networkconfiguation.